fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -829,6 +830,47 @@ public void testIngestWithBinary() throws IOException {
         }
     }
 
+    /**
+     * Ensure that the objects can be created with a Digest header
+     * with a SHA1 sum of the binary content
+     *
+     * @throws IOException in case of IOException
+     */
+    @Test
+    public void testIngestWithBinaryAndChecksum() throws IOException {
+        final HttpPost method = postObjMethod();
+        final File img = new File("src/test/resources/test-objects/img.png");
+        method.addHeader("Content-Type", "application/octet-stream");
+        method.addHeader("Digest", "SHA1=f0b632679fab4f22e031010bd81a3b0544294730");
+        method.setEntity(new FileEntity(img));
+
+        try (final CloseableHttpResponse response = execute(method)) {
```

Comment:
Prefer [the simpler method](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java#L221).